modules/telemetry/src/specific/planetssonification.cpp

```diff
@@ -0,0 +1,779 @@
+    _jupiterProperty.temperatureEnabled.setValue(_jupiterProperty.toggleAll);
+    _jupiterProperty.atmosphereEnabled.setValue(_jupiterProperty.toggleAll);
+    _jupiterProperty.moonsEnabled.setValue(_jupiterProperty.toggleAll);
+}
+void PlanetsSonification::onJupiterSettingChanged() {
+    sendPlanetData(JupiterIndex);
+}
+
+// Saturn
+void PlanetsSonification::onSaturnAllChanged() {
+    _saturnProperty.sizeDayEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.gravityEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.temperatureEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.atmosphereEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.moonsEnabled.setValue(_saturnProperty.toggleAll);
+    _saturnProperty.ringsEnabled.setValue(_saturnProperty.toggleAll);
+}
+void PlanetsSonification::onSaturnSettingChanged() {
+    sendPlanetData(SaturnIndex);
+}
+
+// Uranus
+void PlanetsSonification::onUranusAllChanged() {
+    _uranusProperty.sizeDayEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.gravityEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.temperatureEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.atmosphereEnabled.setValue(_uranusProperty.toggleAll);
+    _uranusProperty.moonsEnabled.setValue(_uranusProperty.toggleAll);
+}
+void PlanetsSonification::onUranusSettingChanged() {
+    sendPlanetData(UranusIndex);
+}
+
+// Neptune
+void PlanetsSonification::onNeptuneAllChanged() {
+    _neptuneProperty.sizeDayEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.gravityEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.temperatureEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.atmosphereEnabled.setValue(_neptuneProperty.toggleAll);
+    _neptuneProperty.moonsEnabled.setValue(_neptuneProperty.toggleAll);
+}
+void PlanetsSonification::onNeptuneSettingChanged() {
+    sendPlanetData(NeptuneIndex);
+}
+
+bool PlanetsSonification::getData(const Camera* camera, int planetIndex,
+                               TelemetryModule::AngleCalculationMode angleCalculationMode,
+                                  bool includeElevation) {
+    double distance = calculateDistanceTo(
+        camera,
+        _planets[planetIndex].name,
+        DistanceUnit::Kilometer
+    );
+
+    double horizontalAngle =
+        calculateAngleTo(camera, _planets[planetIndex].name, angleCalculationMode);
+
+    double verticalAngle = 0.0;
+    if (includeElevation) {
+        verticalAngle = calculateElevationAngleTo(
+            camera,
+            _planets[planetIndex].name,
+            angleCalculationMode
+        );
+    }
+
+    if (std::abs(distance) < std::numeric_limits<double>::epsilon()) {
+        // The scene is likely not yet initialized
+        return false;
+    }
+
+    // Also calculate angle to moons
+    bool shouldSendData = false;
```

Comment:
```suggestion
    bool shouldSendData = false;

    // Also calculate angle to moons
```

Reply:
And why do we also compute the angle too moons?  Could this comment clarify that?